### PR TITLE
Improve doc (tensor values' order)

### PR DESCRIPTION
**Pull request by EmmaRenauld**
This should fix issue https://github.com/dipy/dipy/issues/2673

Adding the information on the order of the tensor values in a few places in both doc and docstrings.

### CODE REVIEW

File: doc/examples/reconst_dti.py [hunk 1]
```diff
@@ -124,7 +124,16 @@
 
 """
 The fit method creates a ``TensorFit`` object which contains the fitting
-parameters and other attributes of the model. For example we can generate
+parameters and other attributes of the model. You can revover the 6 values
```
Comment (skoudoro) [inline doc/examples/reconst_dti.py:127]: typo: revover = recover ?

File: doc/examples/reconst_dti.py [hunk 2]
```diff
@@ -124,7 +124,16 @@
 
 """
 The fit method creates a ``TensorFit`` object which contains the fitting
-parameters and other attributes of the model. For example we can generate
+parameters and other attributes of the model. You can revover the 6 values 
+of the triangular matrix representing the tensor D. By default, in Dipy, values
```
Comment (skoudoro) [inline doc/examples/reconst_dti.py:128]: typo: DIPY uppercase

File: doc/examples/reconst_dti.py [hunk 3]
```diff
@@ -124,7 +124,16 @@
...
+are ordered as (Dxx, Dxy, Dyy, Dxz, Dyz, Dzz). The tensor_val variable defined
```
Comment (skoudoro) [inline doc/examples/reconst_dti.py:129]: `tensor_vals` with a `s` (as you wrote it below) and backticks

**Review by skoudoro — COMMENTED** [review]
Thank you for this PR @EmmaRenauld!

See below for some typo to fix and it will be ready to be merged.

File: doc/examples/reconst_dti.py (hunk 1 above)
Comment (EmmaRenauld) [reply doc/examples/reconst_dti.py:127]: Oups, sorry.

**Comment by skoudoro** [comment]
Thank you @EmmaRenauld !

**Pull request merged**